drivers/gpu/drm/i915/i915_active.c

```diff
@@ -130,6 +131,58 @@ static inline void debug_active_assert(struct i915_active *ref) { }
 
 #endif
 
+#if defined(__FreeBSD__)
+static struct llist_head linux_kmem_cache_list = LLIST_HEAD_INIT();
+
+struct cache_addr {
+	struct linux_kmem_cache *cache;
+	void *addr;
+};
```

Comment:
If this structure is a linked list node, it should include `struct llist_node` as member

Reply:
Added it. Thanks!